packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -109,6 +109,8 @@ export const processPatientDocumentRequest = async (
       );
     }
 
+    patientEvents().emitCanvasIntegration({ id: patientId, cxId, metadata, whType });
```

Comment:
This is supposed to be an event about webhook, not about canvas... like I said before, this is supposed to be agnostic about the listener, so it can be generic.

Reply:
See https://en.wikipedia.org/wiki/Observer_pattern